guides/consumer/basic_setup.md

```diff
@@ -0,0 +1,37 @@
+Consumer basic setup
+====================
+
+# Example
```

Comment:
What is the difference between these two different ways to indicate a headline?

Reply:
Very good question. I do not know. I always use `===` for the title and then `#` for headlines